pages/SavingsTable.js

```diff
@@ -1,16 +1,14 @@
-import AccountTable from './AccountTable';
+import TransactionsTable from './TransactionsTable';
 
-class SavingsTable extends AccountTable {
+class SavingsTable extends TransactionsTable {
 
-    get switchSaving () { return $('//*[contains(text(),"Indiviudal Savings")]//ancestor::div[@class="card-body"]//span[@class="switch-handle"]')}
-
-    get switchChecked () { return $('//*[contains(text(),"Indiviudal Savings")]//ancestor::div[@class="card-body"]//input[@class="switch-input"]')}
+    switch (accountName) { return $(`//*[contains(text(),"${accountName}")]//ancestor::div[@class="card-body"]//span[@class="switch-handle"]`)}
```

Comment:
Not necessary. _SavingsTable_ has access to this selector because it's already in the superclass.